src/modules/ie-objects/helpers/convert-objects-to-csv.ts

```diff
@@ -0,0 +1,14 @@
+import { flatten } from 'flat';
+import Papa from 'papaparse';
+
+import { IeObject } from '../ie-objects.types';
+
+export const convertObjectsToCsv = (objects: Partial<IeObject>[]): string => {
```

Comment:
REQ: test file needs to be written with several usecase to be sure the code works as intended. Example can be found in convert to xml spec file.